index.css

```diff
@@ -114,6 +114,10 @@ button {
     cursor: pointer;
 }
 
+.secondary-btn>a {
```

Comment:
try to avoid complex selectors to make debugging easier in large apps.
Try to apply CSS to single classes as much as possible

Reply:
Thank you for all the suggestions, have made the required changes.
Will work on naming classes more clearly.